webapp/src/components/AcceptBidAssetPage/AcceptBidAssetPage.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { ASSET_TYPES } from 'shared/asset'
+import AcceptBidParcelPage from './AcceptBidParcelPage'
+import AcceptBidEstatePage from './AcceptBidEstatePage'
+import { bidType } from 'components/types'
+
+export default class AcceptBidAssetPage extends React.PureComponent {
+  static propTypes = {
+    id: PropTypes.string.isRequired,
+    bid: bidType,
+    assetType: PropTypes.string.isRequired,
+    isConnected: PropTypes.bool,
+    isLoading: PropTypes.bool.isRequired
+  }
+
+  bidFetched = false
```

Comment:
```suggestion
  isBidFetched = false
```

I know it's not super good english, but to keep consistent